frontend/pages/create_locked_account.tsx

```diff
@@ -31,6 +91,33 @@ const CreateLockedAccount: NextPage = () => {
     }
   }
 
+  const handleSetDate = (event: any) => {
+    try {
+      setStartDate(new Date(event.target.value))
+    }
+    catch (e) {
+      setStartDate(undefined)
+    }
+  }
+
+  const handleSetFirstUnlock = (event: any) => {
+    try {
+      setFirstUnlock(new Date(event.target.value))
+    }
+    catch (e) {
+      setFirstUnlock(undefined)
+    }
+  }
```

Comment:
```suggestion
  const handleSetFirstUnlock = (event: any) => {
      setFirstUnlock(event.target.value ? new Date(event.target.value) : undefined)
  }
```

Reply:
I think `event.target.value` can't be undefined with this type of form, so just made it 
```javascript
  const handleSetFirstUnlock = (event: any) => {
      setFirstUnlock(new Date(event.target.value))
  }
```